content/ekipe/travnik.md

```diff
@@ -0,0 +1,23 @@
++++
+date = "2024-08-31T14:00:00+02:00"

```

Comment:
date mora biti mislim da v letu 2016 (recimo daj kar 2016-10-11T14:30:00+02:00, kot ima zadnja dodana skupina. Gre za nek bug v sortiranju in grupiranju, ki potem za druga leta pokvari layout. Bug, ki se ga ne da nobenemu popraviti.